apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.25.0),
+    tiledb (>= 0.25.0), tiledb (<= 0.25.0.5)
```

Comment:
Yes, correct for release 1.9.* as long as the selected r-universe has a suitable version.